docker-enabler/rootfs/usr/bin/run.sh

```diff
@@ -17,11 +18,19 @@ if ! hass.api.supervisor.ping; then
     hass.die "Cannot reach the HassIO API. How do you expect me to unlock?!"
 fi
 
+if [[ $(hass.api.supervisor.info.version) -gt 128 ]]; then
```

Comment:
There is no need to create backward compatibility for the Supervisor.
The Supervisor updates are pushed, so everybody will be on it within 24 hours.